tapir/shifts/emails/stand_in_found_email.py

```diff
@@ -9,6 +9,9 @@
 
 
 class StandInFoundEmail(TapirEmailBase):
+    optional = True
```

Comment:
I'm not sure this mail should be optional? It's important for the members to know if they need to do extra shifts when they don't show up.

Reply:
I think we need a system where Admins can choose which mail is optional. I set this to optional but enabled_by_default=True, but no hard opinion on this one

Reply:
We could go through the mails together...

Reply:
I don't think we need the admins to set the required/optional options. Once the PR is ready we can ask the Vorstand to go through the list once, but after that it's likely to never change. We can avoid the extra code.

Reply:
I resolve this for now to clear the view but put it under todos